ajax/libs/jquery.ime/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "jquery.ime",
+  "version": "1.0.0",
+  "description": "jQuery based internationalization library",
+  "keywords": [
+    "internationalization",
+    "localization",
+    "ime",
+    "jquery",
+    "l10n"
+  ],
+  "author": {
+    "name": "Santhosh Thottingal",
+    "email": "santhosh.thottingal@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/wikimedia/jquery.ime.git"
+  },
+  "filename": "jquery.ime.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/wikimedia/jquery.ime.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "jquery.ime*"
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "!(README)",
```

Comment:
Please remove this rule. It fetches so many files CDN doesn't need.

Reply:
That was my mistake with pattern the `!(README)` should be included inside rules see here https://github.com/wikimedia/jquery.ime/issues/453